Gems/ScriptCanvas/Code/Editor/Nodes/NodeDisplayUtils.cpp

```diff
@@ -171,6 +171,11 @@ namespace ScriptCanvasEditor::Nodes
             SlotDisplayHelper::DisplayVisualExtensionSlot(graphCanvasEntity->GetId(), extensionConfiguration);
         }
 
+        if (details.m_name.empty())
+        {
+            details.m_name = node->GetNodeName();
```

Comment:
so if no translation, title name will come from behavior context?

Reply:
In this particular case it's only for nodes that derive from ScriptCanvas::Node, Behavior Context nodes are handled properly already